client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -168,6 +172,8 @@ class ListAll extends Component {
 	}
 }
 
+const listAllView = () => recordTracksEvent( 'calypso_domain_management_list_all_view' );
```

Comment:
Doesn't this basically duplicate the `calypso_page_view` event - am I missing something?

Reply:
You are right. I was going to remove this after setting up the page view.